src/components/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './SearchForm.scss';
+
+class SearchForm extends React.Component {
+    handleChangeQuery = (e) => {
+        this.setState({ query: e.target.value });
+    }
+    handleSearch = (e) => {
```

Comment:
You have to add e.preventDefault(), because the page will be refreshed after a form submission.